include/miopen/config.h.in

```diff
@@ -75,6 +75,7 @@
 #cmakedefine EXTRACTKERNEL_BIN "@EXTRACTKERNEL_BIN@"
 #cmakedefine MIOPEN_OFFLOADBUNDLER_BIN "@MIOPEN_OFFLOADBUNDLER_BIN@"
 #cmakedefine MIOPEN_CACHE_DIR "@MIOPEN_CACHE_DIR@"
+#cmakedefine LIBMLIRMIOPEN "@LIBMLIRMIOPEN@"
```

Comment:
This should be prefixed with `MIOPEN_`

Reply:
Will create a new variable, and prefix it with MIOPEN_ in here.